e2e/publish-ports.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
```

Comment:
Nice. Suggestion: it would be awesome add e2e_test --debug flag, which will include this yml automatically, and also would wait for user input before tearing everything down.

Reply:
Implemented